### PR TITLE
Bugfix/load to grid

### PR DESCRIPTION
**Pull request by richadivami**
# TITLE

## Description
Summary of changes

## PR Type
- [ ] Bug Fix
- [ ] Feature
- [ ] Build Fix
- [ ] Testing
- [ ] General Improvement
- [ ] Cherry Pick

## Links
Jira: [Jira issue #](fill-in.org)

### CODE REVIEW

File: app/cdap/components/ConnectionList/index.tsx [hunk 1]
```diff
@@ -51,9 +52,14 @@ const DatasetWrapper = () => {
   const queryParams = new URLSearchParams(loc.search);
   const pathFromUrl = queryParams.get('path') || '/';
   const [loading, setLoading] = useState(false);
+  const [isErrorOnNoWorkspace, setIsErrorOnNoWorkSpace] = useState(false);
 
-  const toggleLoader = () => {
-    setLoading(!loading);
+  const toggleLoader = (value: boolean, isError?: boolean) => {
+    console.log('value, isError', value, isError);
```
Comment (adityadivami) [inline app/cdap/components/ConnectionList/index.tsx]: remove console.log, check all thru the pr and remove everywhere

File: app/cdap/components/SnackbarComponent/index.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+ * the License.
+ */
+import * as React from 'react';
+import Snackbar, { SnackbarOrigin } from '@material-ui/core/Snackbar';
+import SnackbarContent from '@material-ui/core/SnackbarContent';
+import WarningRoundedIcon from '@material-ui/icons/WarningRounded';
+import { Divider } from '@material-ui/core';
+import { useErrorStyles } from './styles';
+
+export interface State extends SnackbarOrigin {
```
Comment (adityadivami) [inline app/cdap/components/SnackbarComponent/index.tsx]: keep interfaces outside in a separate file name types.ts and import and use here

File: app/cdap/components/SnackbarComponent/index.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  open: boolean;
+}
+
+export default function PositionedSnackbar(props) {
```
Comment (adityadivami) [inline app/cdap/components/SnackbarComponent/index.tsx]: use arrow functions everywhere, mandatory

Comment (adityadivami) [reply app/cdap/components/SnackbarComponent/index.tsx]: if it is copied from mui, then convert it to arrow function

File: app/cdap/components/SnackbarComponent/index.tsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+  const handleClick = () => () => {
+    setState({ open: true, ...state });
+  };
+
+  const handleClose = () => {
+    setState({ ...state, open: false });
+    props.handleCloseError();
+  };
+
+  const TransitionComponent = () => {
```
Comment (adityadivami) [inline app/cdap/components/SnackbarComponent/index.tsx]: make a separate component for every component though static or pure html

Comment (adityadivami) [reply app/cdap/components/SnackbarComponent/index.tsx]: have its styles, index and types in a separate component folder

**Review by adityadivami — CHANGES_REQUESTED** [review]
please fix all comments and update me

File: app/cdap/components/ConnectionList/Components/TabLabelCanSample/index.tsx [hunk 5]
```diff
@@ -34,7 +34,7 @@ const TabLabelCanSample = ({
   label: string;
   entity: any;
   initialConnectionId: string;
-  toggleLoader: () => any;
+  toggleLoader: (value: boolean, isError?: boolean) => any;
```
Comment (adityadivami) [inline app/cdap/components/ConnectionList/Components/TabLabelCanSample/index.tsx]: please replace any with somthing appropriate type

File: app/cdap/components/ConnectionList/index.tsx [hunk 6]
```diff
@@ -237,7 +247,7 @@ const DatasetWrapper = () => {
                   value={each.selectedTab}
                   index={index}
                   connectionId={connectionId || ''}
-                  toggleLoader={toggleLoader}
+                  toggleLoader={(value, isError) => toggleLoader(value, isError)}
```
Comment (adityadivami) [inline app/cdap/components/ConnectionList/index.tsx]: try to add types wherever possible

**Review by adityadivami — CHANGES_REQUESTED** [review]

File: app/cdap/components/ConnectionList/index.tsx (hunk 1 above)
Comment (richadivami) [reply app/cdap/components/ConnectionList/index.tsx]: Done

File: app/cdap/components/SnackbarComponent/index.tsx (hunk 2 above)
Comment (richadivami) [reply app/cdap/components/SnackbarComponent/index.tsx]: Done

File: app/cdap/components/SnackbarComponent/index.tsx (hunk 3 above)
Comment (richadivami) [reply app/cdap/components/SnackbarComponent/index.tsx]: Done

File: app/cdap/components/SnackbarComponent/index.tsx (hunk 4 above)
Comment (richadivami) [reply app/cdap/components/SnackbarComponent/index.tsx]: Done

File: app/cdap/components/ConnectionList/Components/TabLabelCanSample/index.tsx (hunk 5 above)
Comment (richadivami) [reply app/cdap/components/ConnectionList/Components/TabLabelCanSample/index.tsx]: Done

File: app/cdap/components/ConnectionList/index.tsx (hunk 6 above)
Comment (richadivami) [reply app/cdap/components/ConnectionList/index.tsx]: Done

**Review by adityadivami — APPROVED** [review]

**Pull request merged**